chips/x86_q35/src/interrupt_service.rs

```diff
@@ -0,0 +1,20 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2025.
+
+use kernel::platform::chip::InterruptService;
+
+/// A default interrupt service that does nothing and reports no handled IRQs.
+///
+/// Boards can use this as a placeholder when they don't have board-specific
+/// devices that need custom IRQ handling. All interrupts are reported as
+/// unhandled by returning `false`.
+#[derive(Copy, Clone, Debug)]
+pub struct PcDefaultPeripherals;
```

Comment:
Hrm... kind of seems like this should be in the kernel crate? There isn't anything `x86_q35`-specific about this, is there?

Reply:
I don't necessarily _mind_ it being here for now, and we can move it if/when anyone wants to use it on a different platform. My first guess would be other, non q35, x86 chips that may end up copy-paste-modifying much of this crate as a starting point.